docs/dev/setup/bamboo-bitbucket-jira.rst

```diff
@@ -35,11 +35,28 @@ Docker-Compose
 Before you start the docker-compose, check if the bamboo version in the
 ``build.gradle`` (search for ``com.atlassian.bamboo:bamboo-specs``) is
 equal to the bamboo version number in the Dockerfile of bamboo stored in
-``src/main/docker/bamboo/Dockerfile``. If the version number is not
-equal adjust the version number in the Dockerfile.
+``src/main/docker/bamboo/Dockerfile`` or ``src/main/docker/bamboo/swift/Dockerfile``.
```

Comment:
I have a general question: Why not simply create a single Dockerfile that includes all of the programming languages? It would be much easier to maintain and develop since we don't have different docker containers

Reply:
it was a request by @krusche to split those and have one very simple Dockerfile, for the general tasks. 
Maybe, when we have more Dockerfiles (at some point) we could combine those into one that contains them all, next to each separate one.

Reply:
Ah okay!